xinference/web/ui/src/scenes/launch_model/index.js

```diff
@@ -11,6 +11,7 @@ import LaunchCustom from './launchCustom'
 import LaunchEmbedding from './launchEmbedding'
 import LaunchLLM from './launchLLM'
 import LaunchRerank from './launchRerank'
+import LaunchText2Image from "./launchText2Image";
```

Comment:
launch model use the model type as part of the file name, e.g. `imageCard.js`, `launchImage.js`.